ols/src/query_helpers/question_validator.py

```diff
@@ -21,9 +21,8 @@ class QuestionValidator(QueryHelper):
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Initialize the QuestionValidator."""
-        # TODO: OLS-447 Refactor code that pass parameters into LLM in QuestionValidator
-        llm_params = {"max_new_tokens": 4}
-        super().__init__(*args, **dict(kwargs, llm_params=llm_params))
+        generic_llm_params = {"max_new_tokens": 4}
```

Comment:
replace w/ generic parameter constant if you agree to creating one.

Reply:
+1

Reply:
updated @bparees